.taskcluster.yml

```diff
@@ -254,13 +285,25 @@ tasks:
               features:
                 taskclusterProxy: true
               maxRunTime: 3600
-              image: "mozilla/taskboot:0.2.5"
+              image: "mozilla/taskboot:0.2.7"
               env:
                 TASKCLUSTER_SECRET: project/relman/rust-code-analysis/deploy
               command:
                 - "/bin/sh"
                 - "-cx"
-                - "taskboot github-release mozilla/rust-code-analysis ${version} --asset ${linux_artifact}:public/${linux_artifact} ${windows_artifact}:public/${windows_artifact}"
+                - "git config --global user.email you@example.com &&
```

Comment:
Nit: use `moz-tools-bot@moz.tools`, even if it's fake, it's nicer than example.com :P